src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -43,6 +63,48 @@ public Book.Book CurrentBook
 			set { ReadersHandler.CurrentBook = value; }
 		}
 
+		Dictionary<string, string> _fakeTempFiles = new Dictionary<string, string>();
```

Comment:
I can't tell what this is. I can see that the key to the dictionary is a url, but not yet what the value is. I'll find it eventually here, but maybe we could name it something like "URLTo____Dictionary" where the blank is what the value is. And give it a comment. And move it to the top of the class.